Documentation/README.md

```diff
@@ -0,0 +1,21 @@
+
+# Documentation
+
+<br>
+
+## Assembly
+
+*How to build this project.*
+
+- **[Wireless]**
+
+- **[Wired]**
```

Comment:
These links are pretty important, I'd move them back to the top-level readme page.

Reply:
Sure